contract/contracts/DUSKMigration.sol

```diff
@@ -12,20 +12,41 @@ interface IERC20 {
 
 contract DUSKMigration {
     IERC20 public duskToken;
```

Comment:
```suggestion
    IERC20 public immutable duskToken;
```

Can be made `immutable` to make the migration a little bit cheaper (gas wise).

Reply:
@nekonimous thank you! I'm pretty surprised by how much of a difference it makes, given the compiler could've theoretically made assumptions about the immutable nature of duskToken itself.

Here's a comparison of gas usage in repeated calls to the contract:
| Contract version         | Gas Usage | Transaction Cost |
|--------------------------|-----------|------------------|
| [5627c29cfdc7e9cd88f5a8c6cb9d908c967812eb](https://github.com/dusk-network/dusk-migration/blob/5627c29cfdc7e9cd88f5a8c6cb9d908c967812eb/contract/contracts/DUSKMigration.sol)                   | 59707 gas | 51919 gas        |
| LUX conversion            | 60434 gas | 52551 gas        |
| LUX conversion immutable  | 57982 gas | 50419 gas        |